packages/transaction-controller/src/TransactionController.ts

```diff
@@ -1109,6 +1115,84 @@ export class TransactionController extends BaseController<
     return this.nonceTracker.getNonceLock(address);
   }
 
+  async approveTransactionsWithSameNonce(
+    listOfTxParams: TransactionParams[] = [],
+  ) {
```

Comment:
Should we add a return type? I assume just `Promise<string>`?

Reply:
Added in https://github.com/MetaMask/core/pull/1961/commits/717d37954756c7246b5dbb18759f2fb74c53a8a3